Jenkinsfile

```diff
@@ -478,55 +528,92 @@ pipeline {
 
 def makeTarget(context, target, clean = true){
   withGithubNotify(context: "${context}") {
-    sh(label: "Make ${target}", script: "make ${target}")
+    withBeatsEnv(){
+      sh(label: "Make ${target}", script: """
+        eval "\$(gvm use ${GO_VERSION} --format=bash)"
```

Comment:
Just curious, why do we need the `eval` here, but not for the `make` command below?